src/SDKs/Azure.Base/data-plane/Azure.Base/Http/Pipeline/ExponentialRetryPolicy.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+
+namespace Azure.Base.Http.Pipeline
+{
+    internal class ExponentialRetryPolicy : RetryPolicy
+    {
+        private readonly int[] _retriableCodes;
+
+        private readonly Func<Exception, bool> _exceptionFilter;
```

Comment:
Can we rename it to make it clear whether this is including or excluding? e.g _shouldRetryException

Reply:
Also, should we have ShouldRetryException virtual method? just like for status codes as opposed to one method for both exceptions and codes?

Reply:
I'll add ShouldRetryException